app/src/routes/constants.js

```diff
@@ -92,7 +92,7 @@ export const pageRendering = {
   PROJECT_FILTERS_PAGE: { component: FiltersPage, layout: AppLayout },
   [LAUNCHES_PAGE]: { component: LaunchesPage, layout: AppLayout },
   PROJECT_LAUNCHES_PAGE: { component: LaunchesPage, layout: AppLayout },
-  PROJECT_MEMBERS_PAGE: { component: ProjectMembersPageContainer, layout: AppLayout },
```

Comment:
Please remove the `ProjectMembersPageContainer` and all related components in case no needed anymore.

Reply:
Done.